frontend/src/cmps/popovers/popover-leave-board.jsx

```diff
@@ -0,0 +1,47 @@
+import { useNavigate } from "react-router-dom";
+import { useDispatch, useSelector } from "react-redux";
+import _ from "lodash";
+
+import { PopoverCmp } from "./popover-cmp";
+
+import { userService } from "../../services/user.service";
+import { onEditBoard } from "../../store/actions/board.actions";
+import { closePopover } from "../../store/actions/app.actions";
+
+export const PopoverLeaveBoard = ({ elPos, handleClose }) => {
+  const board = useSelector((state) => state.boardReducer.board);
+  const loggedInUser = userService.getLoggedinUser();
+
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  const onLeaveBoard = (ev) => {
+    ev.preventDefault();
+    if (
+      board.boardMembers.some(
+        (member) => member.fullname === loggedInUser.fullname
+      )
+    ) {
+      let boardToSave = _.cloneDeep(board);
+      const memberIdx = boardToSave.boardMembers.findIndex(
```

Comment:
check if needed to also remove user from all cards.

Reply:
done